src/core_plugins/timelion/public/panels/timechart/schema.js

```diff
@@ -284,7 +332,14 @@ export default function timechartFn(Private, config, $rootScope, timefilter, $co
 
           if (_.get($scope.plot.getData(), '[0]._global.legend.showTime', true)) {
             legendCaption = $('<caption class="timelionLegendCaption"></caption>');
+            legendCaption.html(emptyCaption);
             canvasElem.find('div.legend table').append(legendCaption);
+
+            // legend has been re-created. Apply focus on legend element when previously set
+            if (focusedSeries) {
```

Comment:
`focusedSeries || focusedSeries === 0` otherwise we loose the focus when focusing the first element, because `0 == false`.